test/torchaudio_unittest/datasets/libritts_test.py

```diff
@@ -11,42 +9,54 @@
     normalize_wav,
 )
 
+from torchaudio.datasets.libritts import LIBRITTS
+
+_UTTERANCE_IDS = [
+    [19, 198, '000000', '000000'],
+    [26, 495, '000004', '000000'],
+]
+_ORIGINAL_TEXT = 'this is the original text.'
+_NORMALIZED_TEXT = 'this is the normalized text.'
+
+
+def get_mock_dataset(root_dir):
```

Comment:
Can you make `get_mock_dataset` return all the expected sample data, i.e. texts and utterance IDs, so that test suites do not need to refer to global variables?